src/nfv_test_api/main.py

```diff
@@ -113,6 +114,37 @@ def add_namespace(namespace):
     return jsonify({})
 
 
+@app.route("/<namespace>/", methods=["DELETE"])
+def delete_namespace(namespace):
+    if namespace is None or len(namespace) == 0:
+        raise exceptions.ServerError(f"Invalid namespace {namespace}")
+
+    interfaces: List = list_interfaces(namespace).get_json()
+    untagged: List = [
+        interface
+        for interface in interfaces
+        if "." not in interface and interface != "lo"
+    ]
+    if untagged:
+        raise exceptions.ServerError(
```

Comment:
Is this right or should this be a 4xx return code? Can I attach a message to those?

Reply:
I think 4xx might be more appropriate, but for consistency with errors at other endpoints (e.g., line 155, which could also be a 400), it can also stay like this